README.md

```diff
@@ -56,23 +54,233 @@ druid-operator$ kubectl apply -f examples/tiny-cluster-zk.yaml
 # deploy druid cluster spec
 druid-operator$ kubectl apply -f examples/tiny-cluster.yaml
 ```
+###  Druid CR Spec
```

Comment:
does it make sense to move this stuff into a "docs/" directory and add right links here instead. in that world, it would be easier to evolve the documentation into a multi page thing as it grows.

Reply:
agreed

Reply:
@himanshug in the docs directory shall we have seperate files.md for each link or combined one ?
I guess seperate would be better.